docker-compose.yml

```diff
@@ -13,6 +13,8 @@ services:
   db:
     image: "mysql:5"
     environment:
+      XWIKI_VERSION: 7.4.3
+      MYSQL_DRIVER_VERSION: 5.1.38
```

Comment:
I'm curious: have you tried it? I'm asking since I don't see how it could work :)